src/Uno.UI/UI/Xaml/HitTestability.cs

```diff
@@ -34,4 +34,38 @@ internal enum HitTestability
 	internal record struct StalePredicate(PredicateOfUIElement Method, string Name);
 
 	internal delegate bool PredicateOfUIElement(UIElement element);
+
+	internal static class GetHitTestabilityExtensions
+	{
+		/// <summary>
+		/// Wrap the given hitTest delegate to exclude (i.e. consider as <see cref="HitTestability.Collapsed"/>) the provided element.
+		/// </summary>
+		/// <param name="hitTest">The hit-testing delegate to wrap.</param>
+		/// <param name="element">The element that should be considered as <see cref="HitTestability.Collapsed"/>)</param>
+		/// <returns></returns>
+		internal static GetHitTestability Except(this GetHitTestability hitTest, UIElement element)
+		{
+			GetHitTestability hitTestExceptElement = default!;
```

Comment:
```suggestion
		{
			if (hitTest == null) throw new ArgumentNullException(nameof(hitTest));
			if (element == null) throw new ArgumentNullException(nameof(element));
			
			GetHitTestability hitTestExceptElement = default!;
```
Maybe best to add error handling to avoid runtime exceptions here no?

Reply:
It's internal and nullable annotations has been enabled here, there is no gain to do so.